wordpress-mobile/WordPress-Android
# RI: Introduce "Welcome to Reader" card

Parent Issue: #12028

This PR adds
- "Welcome to Reader" card 
- Item decorator

to new discover feed.

"Welcome to Reader" card is hidden when list is refreshed.

 Phone Portrait Light |  Phone Phone Dark
---------------| -----------------
![phone_light_portrait](https://user-images.githubusercontent.com/1405144/90399087-04f52380-e0b8-11ea-9499-84f64ab0df89.png) | ![phone_portrait](https://user-images.githubusercontent.com/1405144/90399061-f7d83480-e0b7-11ea-97b4-05a6362dd2d2.png)

Phone Landscape Light | Phone Landscape Dark
---------------| -----------------
 ![device-2020-08-20-144404](https://user-images.githubusercontent.com/1405144/90751510-d95e7d00-e2f3-11ea-8041-69ff5395f6e4.png)| ![device-2020-08-20-144444](https://user-images.githubusercontent.com/1405144/90751463-c8ae0700-e2f3-11ea-86cb-d0948516ccc8.png)


Tablet Portrait Light | Tablet Landscape Light
---------------| -----------------
![tablet_portrait_light](https://user-images.githubusercontent.com/1405144/90399796-2c002500-e0b9-11ea-9b1e-0929024555ba.png)| ![tablet_landscape_light](https://user-images.githubusercontent.com/1405144/90399817-328e9c80-e0b9-11ea-96c5-00fe91b587ae.png)

To test:
Prerequisite: RI FF is enabled

1. Clean app data
2. Launch app, login and go to Reader -> Discover tab
3. Notice that "Welcome Banner" card is shown
4. Pull to refresh
5. Notice that  "Welcome Banner" card is gone

PR submission checklist:
